packages/Python/lldbsuite/test/lang/swift/swift_interface_shared/TestSwiftInterfaceNoDebugInfo.py

```diff
@@ -30,12 +30,14 @@ class TestSwiftInterfaceNoDebugInfo(TestBase):
     mydir = TestBase.compute_mydir(__file__)
 
     @swiftTest
+    @skipIfLinux # rdar://49662256
```

Comment:
Can you change this to 
```
    @expectedFailureAll(oslist=["linux"],bugnumber="rdar://49662256")
```
This way it's less likely that we forget to remove the skip later on.

Reply:
Sure, definitely. Should I also update this on master?